src/plugins/telemetry/common/constants.ts

```diff
@@ -32,7 +32,7 @@ export const LOCALSTORAGE_KEY = 'telemetry.data';
 /**
  * Link to Advanced Settings.
  */
-export const PATH_TO_ADVANCED_SETTINGS = '/app/management/kibana/settings';
+export const PATH_TO_ADVANCED_SETTINGS = 'management/kibana/settings';
```

Comment:
Hum, the management app is accessible at `/app/management`, not `/management` 🤔

https://github.com/elastic/kibana/blob/b217dbf001b71eeba739aae7dab93e8f05750922/src/plugins/management/public/plugin.ts#L100-L101

Are we sure this did not break anything?

Reply:
It's only used here: https://github.com/elastic/kibana/blob/ac9ed3ae48de3d918d01a83c5bfbf6b70c6d5266/src/plugins/telemetry/public/components/opt_in_message.tsx#L98

I noticed that it works either way. And it breaks with `export const PATH_TO_ADVANCED_SETTINGS = 'app/management/kibana/settings';` (mind the missing heading `/`).

I'll revert the change because, IMO, it feels safer to include `/app` :)

Reply:
Pushed in [`a6d34c8` (#159406)](https://github.com/elastic/kibana/pull/159406/commits/a6d34c8c32bacc2c4130625df57778c35dc1ad92)